catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -352,8 +355,7 @@ private String durationToString(final Duration duration) {
   }
 
   private DefaultExecuteResultHandler executeFFmpeg(
-      final CommandLine command, final int timeoutSeconds, final PumpStreamHandler streamHandler)
-      throws IOException {
```

Comment:
❓ It looks like this IOException was being bubbled up. Do we need to do an instance of check on the privileged action caused by exception and rethrow if it is an IOException?